env.daint.sh

```diff
@@ -15,3 +15,9 @@ export CUDA_HOME=$(echo $NVCC_PATH | sed -e "s/\/bin\/nvcc//g")
 export LD_LIBRARY_PATH=$CUDA_HOME/lib64:$LD_LIBRARY_PATH
 export MPICH_RDMA_ENABLED_CUDA=1
 
+# since gridtools does not play nice with gcc 8.3 we switch to 8.1
+module switch gcc/8.3.0 gcc/8.1.0
+
+# the eve toolchain requires a clang-format executable, we point it to the right place
+export CLANG_FORMAT_EXECUTABLE=`/project/s1053/install/venv/vcm_1.0/bin/clang-format`
```

Comment:
Should these be single quotes rather than backticks?
```bash
export CLANG_FORMAT_EXECUTABLE='/project/s1053/install/venv/vcm_1.0/bin/clang-format'
```
Also trailing newline, but not a big deal...